specifications/catalog/catalog.protocol.md

```diff
@@ -56,17 +54,31 @@ This section describes how the IDS Information Model maps to DCAT resources.
 
 An `Asset Entry` is a [DCAT Dataset](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset) with the following attributes:
 
-#### 3.1.1 dcat:haPolicy
+#### 3.1.1 odrl:hasPolicy
 
+a)
 An asset entry Dataset may have 0..N `hasPolicy` attributes that contain an ODRL `Offer` defining the usage control policy associated with the asset. Offers must NOT contain any
 target attributes. The target of an offer is the asset associated with the containing asset entry. If an entry is has no associated policies (hasPolicy does not contain any offer
 entries), the asset is does not have any usage control policy.
 
+b)
+An asset entry Dataset shall not contain an ODRL `Offer`. Offers can only appear at `Distributions`. 
```

Comment:
Is that what we agreed on in Munich? I must have missed that...

Reply:
I agree, we didn't agree to that ;-)

Reply:
No we didn't :-)

I have removed the sentence, also based on our call today. 

More general point: How shall we work with alternative statements? My approach was marking them with 'a)' and 'b)', do you have a better idea?